dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf

```diff
@@ -93,6 +93,9 @@ server=www.ovh.com, \
 protocol={{account.service}}, \
 apikey={{account.username}}, \
 secretapikey={{account.password}}, \
+{%      elif account.service == 'digitalocean' %}
```

Comment:
can you try to retain alphabetical ordering here? it's already a bit difficult to follow this part of the template with the duplication (ideally it's the same as cloudflare or others).

Reply:
you mean something like that?
```patch
diff --git a/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf b/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf
index 6c2cf332..333d478a 100644
--- a/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf
+++ b/dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf
@@ -36,7 +36,7 @@ use=cmd, cmd="/usr/local/opnsense/scripts/ddclient/checkip -t {{account.force_ss
 {%      if account.service == 'custom' %}
 protocol={{account.protocol}}, \
 server={{account.server}}, \
-{%      elif account.service == 'cloudflare' %}
+{%      elif account.service in ['cloudflare', 'digitalocean'] %}
 protocol={{account.service}}, \
 zone={{account.zone}}, \
 {%      elif account.service == 'cloudns' %}
@@ -93,9 +93,6 @@ server=www.ovh.com, \
 protocol={{account.service}}, \
 apikey={{account.username}}, \
 secretapikey={{account.password}}, \
-{%      elif account.service == 'digitalocean' %}
-protocol={{account.service}}, \
-zone={{account.zone}}, \
 {%      else %}
 protocol={{account.service}}, \
 {%      endif %}

```

Reply:
yeah, perfect ❤️